File: Java/AlarmsObjectOriented/src/main/java/alarms/LoudAlarm.java
package alarms;

import interfaces.Alarm;
import adapters.SirensService;

public class LoudAlarm implements Alarm {
  public void trigger() {
    SirensService.playLoudSound();
  }

  public void disable() {
    SirensService.stopPlayingLoudSound();
  }

  public void dump() {
    System.out.println("{ Playing loud sound }");
  }
}

File: CSharp/DomainSpecificLanguages/DomainSpecificLanguages/AlarmsWithNestedFunctions/Alarms/LoudAlarm.cs
using System;

namespace AlarmsWithNestedFunctions.Alarms
{
  public class LoudAlarm : Alarm
  {
    public void Trigger()
    {
      Console.WriteLine("Playing very loud sound!");
    }

    public void Disable()
    {
      Console.WriteLine("Not playing very loud sound anymore");
    }
  }
}